interface/admin.py

```diff
@@ -44,3 +45,24 @@ def download_submissions(self, request, queryset):
             return FileResponse(review_zip)
 
     download_submissions.short_description = 'Download submissions for review'
+
+
+@admin.register(Submission)
+class SubmissionAdmin(admin.ModelAdmin):
+    actions = ['rerun_submissions']
+
+    def rerun_submissions(self, request, submissions):
+        for old_submission in submissions:
+            new_submission = Submission.objects.create(
```

Comment:
Ah, you're creating new submissions, but I think that's overkill, and counterproductive, because a submission maps to a user action (please take this zip and evaluate it), makes no sense to create another one. Just re-deploy it.